src/commands/force/user/password/generate.ts

```diff
@@ -26,6 +27,14 @@ export class UserPasswordGenerateCommand extends SfdxCommand {
       char: 'o',
       description: messages.getMessage('flags.onBehalfOf'),
     }),
+    length: flags.number({
```

Comment:
we should use the `min` and `max` options on `flags.number` to do the validation